charts/gardener-extension-runtime-gvisor/values.yaml

```
@@ -1,6 +1,6 @@
 image:
   repository: eu.gcr.io/gardener-project/gardener/extensions/runtime-gvisor
-  tag: latest
+  tag: 1.0.0
```

```suggestion
  tag: latest
```

In the Helm chart `latest` is better as we don't want to maintain the tag here.